RocketControlUnitGUI/src/routes/+page.svelte

```diff
@@ -410,6 +410,43 @@
 		});
 	}
 
+	let wasLiveAtAnyPoint = false;
+
+	async function pollIgnitors() {
+		if (box1_display === 'LIVE' || box2_display === 'LIVE') {
+			return wasLiveAtAnyPoint = true;
```

Comment:
Should do what we expect it to, but returning `=` is a little less clear than just assigning it

Reply:
done